Pass foreign to AM_INIT_AUTOMAKE so it doesn't enforce GNU strictness.

Upstream-Status: Pending
Signed-off-by: Ross Burton <ross.burton@intel.com>

diff --git a/configure.ac b/configure.ac
index e602fd3..e075b86 100644
--- a/configure.ac
+++ b/configure.ac
@@ -29 +29 @@ AC_CONFIG_SRCDIR([decoder.h])
-AM_INIT_AUTOMAKE
+AM_INIT_AUTOMAKE([foreign])
